tests/test_annotations.py

```diff
@@ -56,6 +69,46 @@ def test_duplicate_headers(self):
         with self.assertRaises(ValueError):
             dup_headers.preprocess()
 
+    def test_coerce_group_values(self):
+        headers = ["NAME", "cell_type", "organism_age"]
+        annot_types = ["TYPE", "group", "numeric"]
+        expected_dtypes = {"NAME": np.str, "cell_type": np.str}
+        dtypes = Annotations.coerce_group_values(headers, annot_types)
+        self.assertEqual(expected_dtypes, dtypes)
+
+    def test_convert_header_to_multiIndex(self):
```

Comment:
this seems like an implementation detail that doesn't need testing -- that function is two lines of standard pandas usage